mlir/lib/Dialect/Linalg/Transforms/WinogradConv2D.cpp

```diff
@@ -837,9 +837,25 @@ Value outputTransform(RewriterBase &rewriter, Location loc, Value value,
     Value widthOffset =
         builder.create<affine::AffineApplyOp>(loc, affineMap, tileWIter);
 
+    // Handling bias.
```

Comment:
Sorry if I miss something but what bias are you actually handling here?

Reply:
The output tensor is padded, pre-initialized with bias values, and supplied to output transform op as an output parameter. However the pre-initialized values will be discarded later when the output transformation actually happens. The patch fixes the issue.

Reply:
Ok so this is DPS out init value handling from what I gather and not "bias" per se. Practically of course someone can init the output with the bias values etc.
Would suggest rephrasing it a bit.